jiant/trainer.py

```diff
@@ -335,7 +340,9 @@ def _setup_training(
                 biggest_batch_first=True,
             )
             task_info["iterator"] = iterator
-            task_info["tr_generator"] = iterator(task.train_data, num_epochs=None)
+            task_info["tr_generator"] = iterator(
```

Comment:
The naming seems a bit confusing here. If you're calling 'get_instance_generator', shouldn't that return something that can be used as a 'generator'?

Reply:
Thanks — resolved some naming confusion in my changes w/ 72a5bfe.